.github/workflows/rw-fossa-update-attribution-file.yml

```diff
@@ -0,0 +1,69 @@
+# (C) 2024 GoodData Corporation
+
+name: FOSSA ~ Update attribtion file
+on:
+    workflow_call:
+        inputs:
+            source-branch:
+                required: true
+                description: "The name of the source branch"
+                type: string
+
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.source-branch }}
+          token: ${{ secrets.TOKEN_GITHUB_YENKINS_ADMIN }}
+      - name: Add repository to git safe directories to avoid dubious ownership issue
+        run: git config --global --add safe.directory $GITHUB_WORKSPACE
+      - name: Config user
+        run: |
+            git config --global user.email "git-action@gooddata.com"
+            git config --global user.name "git-action"
+      - name: Install fossa
+        run: |
+          curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | sudo bash
+      - name: dummy package.json
+        run: |
+          echo '{}' > common/config/rush/package.json
```

Comment:
I believe this `hack` deserves some explicit comment that it's needed, otherwise fossa wont work

Reply:
fixed